docs/querying/sql.md

```diff
@@ -398,6 +398,17 @@ at execution time. To use dynamic parameters, replace any literal in the query w
 corresponding parameter value when you execute the query. Parameters are bound to the placeholders in the order in
 which they are passed. Parameters are supported in both the [HTTP POST](../api-reference/sql-api.md) and [JDBC](../api-reference/sql-jdbc.md) APIs.
 
+Druid supports double and null values in arrays for dynamic queries. For example:
```

Comment:
It could help to introduce the example in terms of what it does (I had to double check against the ARRAY_CONTAINS [description](https://druid.apache.org/docs/latest/querying/sql-array-functions)). This is my interpretation, but I'm not sure if this is correct:

The query returns `doubleArrayColumn` when the reference array `[-25.7, null, 36.85]` contains all elements of the value of `doubleArrayColumn`.

Reply:
updated